README.md

```diff
@@ -122,6 +122,12 @@ You can deploy this project using:
 - Twitter: [@Yidnekassahun](https://twitter.com/Yidnekassahun)
 - LinkedIn: [@yidne kassahun](https://www.linkedin.com/in/yidnekachew-kassahun-2b817a24b/)
 
+👤👤 **Yosef Geda**
+
+- GitHub: [@YosefGeda](https://github.com/Yosefgeda)
+- Twitter: [@Yosefgeda](https://twitter.com/Yosefgeda)
+- LinkedIn: [@yosefgeda](https://www.linkedin.com/in/yosefgeda/)
```

Comment:
Please, kindly fix the broken `Twitter` profile link, Let's avoid broken links in our readme for a professional repository. 🤗🛠

![image](https://github.com/Yidnekachew-cmd/vet-clinic/assets/78251927/a525a3f3-343b-4884-ad21-f0ed2c75f7f7)